### PR TITLE
Putevent

### PR DESCRIPTION
**Pull request by AsifIslam11197**
Added a put for the events/{id}/edit

### CODE REVIEW

**Comment by AsifIslam11197** [comment]
@JefStat if you please do the honors of reviewing

**Comment by JefStat** [comment]
Lol 
```
@AsifIslam11197	REVERT			e1ea424
 @AsifIslam11197	UN-REVERT			d60cb36 
```

File: src/events.js [hunk 1]
```diff
@@ -39,6 +39,17 @@ var _create = function(ev) {
     });
 };
 
+var _editEvent = function(id, body) {
+  var url = URL.resolve(eventsUrl(this.host), id);
```
Comment (JefStat) [inline src/events.js]: make these 2 URL.resolve lines into a single one.

**Comment by AsifIslam11197** [comment]
@JefStat updated

File: src/events.js [hunk 2]
```diff
@@ -40,8 +40,7 @@ var _create = function(ev) {
 };
 
 var _editEvent = function(id, body) {
```
Comment (JefStat) [inline src/events.js]: just saw the usage in the other pr rename this to `_edit`

**Pull request merged**